01_P/P_2_2_4_02/sketch.js

```diff
@@ -34,16 +35,13 @@ var r = [maxCount]; // radius
 
 var drawGhosts = false;
 
-
 function setup() {
   createCanvas(800, 800);
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).